ProcessManager.php

```diff
@@ -1184,6 +1185,10 @@ protected function newInstance($port)
 
 require_once $dir . '/functions.php';
 
+if(!pcntl_enabled()){
```

Comment:
First, I'm not sure this is the best place to check, but I could not find the better. Please, add suggestions.